empiric-package/empiric/core/base_client.py

```diff
@@ -56,85 +70,12 @@ async def _fetch_base_contracts(self):
                 self.account_contract_address, self.client
             )
 
-    async def get_eth_balance(self):
-        if self.network == "testnet":
-            eth_address = (
-                0x049D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7
-            )
-        else:
-            raise NotImplementedError(
-                "EmpiricBaseClient.get_eth_balance: Unknown network type"
-            )
-
-        with open(path.join(path.dirname(__file__), "abi/ERC20.json"), "r") as f:
-            erc20_abi = json.load(f)
-        contract_data = ContractData.from_abi(eth_address, erc20_abi)
-        balance_of_abi = [a for a in erc20_abi if a["name"] == "balanceOf"][0]
-        balance_of_function = ContractFunction(
-            "balanceOf", balance_of_abi, contract_data, self.client
-        )
-
-        result = await balance_of_function.call(self.account_contract_address)
-
-        return result.balance
-
-    async def send_transaction(
-        self, to_contract, selector_name, calldata, max_fee=None
-    ):
-        return await self.send_transactions(
-            [(to_contract, selector_name, calldata)], max_fee
-        )
-
-    async def send_transactions(self, calls, max_fee=None):
-        # Format data for submission
-        call_array = []
-        offset = 0
-        for i in range(len(calls)):
-            call_array.append(
-                {
-                    "to": calls[i][0],
-                    "selector": get_selector_from_name(calls[i][1]),
-                    "data_offset": offset,
-                    "data_len": len(calls[i][2]),
-                }
-            )
-            offset += len(calls[i][2])
-        calldata = [x for call in calls for x in call[2]]
-
-        # Estimate fee
```

Comment:
How come this is not needed anymore?

Reply:
It happens behind the scenes within the `execute` call. Generally better to use the SDK than try to reinvent the wheel ourselves

Reply:
(was broken in a previous version, but I flagged on their discord and they fixed, now fixing in our downstream dep)